[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:30,583 INFO [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:30,583 INFO [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:219] - [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。] Unexpected exception during request: [null, java.io.IOException: 远程主机强迫关闭了一个现有的连接。, java.io.IOException: 远程主机强迫关闭了一个现有的连接。]
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:30,792 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:30,792 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:31,884 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:31,884 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:36,387 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:36,387 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:37,383 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:37,390 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:41,588 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:41,588 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:42,593 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:42,594 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:46,788 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:46,788 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:47,793 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:47,799 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:51,991 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:51,988 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:52,994 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:54:53,011 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:01,189 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:01,289 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:02,194 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:02,293 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:18,591 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:18,691 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:19,612 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:19,694 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:49,694 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:49,794 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:50,698 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:55:50,798 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:20,796 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:20,896 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:21,807 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:21,898 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:51,899 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:51,999 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:52,901 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:56:53,002 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:23,002 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:23,102 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:24,006 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:24,102 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:54,104 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:54,204 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:55,107 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:57:55,217 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:25,207 INFO [lettuce-eventExecutorLoop-1-4][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:25,307 INFO [lettuce-eventExecutorLoop-1-5][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:26,209 WARN [lettuce-nioEventLoop-4-3][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:26,309 WARN [lettuce-nioEventLoop-4-4][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:56,310 INFO [lettuce-eventExecutorLoop-1-6][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:56,410 INFO [lettuce-eventExecutorLoop-1-7][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:57,324 WARN [lettuce-nioEventLoop-4-5][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:58:57,414 WARN [lettuce-nioEventLoop-4-6][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:27,413 INFO [lettuce-eventExecutorLoop-1-1][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:27,513 INFO [lettuce-eventExecutorLoop-1-8][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:28,417 WARN [lettuce-nioEventLoop-4-7][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:28,514 WARN [lettuce-nioEventLoop-4-8][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:58,516 INFO [lettuce-eventExecutorLoop-1-2][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:58,616 INFO [lettuce-eventExecutorLoop-1-3][AbstractInternalLogger.java:173] - Reconnecting, last destination was localhost:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:59,517 WARN [lettuce-nioEventLoop-4-1][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
[${project.name}][${env:SERVER_PORT}] 2019-12-28 10:59:59,628 WARN [lettuce-nioEventLoop-4-2][AbstractInternalLogger.java:176] - Cannot reconnect: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: localhost/127.0.0.1:6379
